server/middleware/populateCurrentUser.test.ts

```diff
@@ -0,0 +1,74 @@
+import { createMock } from '@golevelup/ts-jest'
+import type { Request, Response } from 'express'
+import jwt from 'jsonwebtoken'
+import type { UserService } from '../services'
+import type { UserDetails } from '../services/userService'
+import populateCurrentUser from './populateCurrentUser'
+
+function createToken(authorities: string[]) {
+  const payload = {
+    user_name: 'USER1',
+    scope: ['read', 'write'],
+    auth_source: 'nomis',
+    authorities,
+    jti: 'a610a10-cca6-41db-985f-e87efb303aaf',
+    client_id: 'clientid',
+  }
+
+  return jwt.sign(payload, 'secret', { expiresIn: '1h' })
+}
+
+describe('populateCurrentUser', () => {
+  function createResWithToken({ authorities }: { authorities: string[] }): Response {
```

Comment:
v v minor but while we're here could change `createToken` and `createResWithToken` to arrow functions for style consistency, and have both within scope of `populateCurrentUser`

```suggestion

  const createToken = (authorities: string[]) => {
    const payload = {
      user_name: 'USER1',
      scope: ['read', 'write'],
      auth_source: 'nomis',
      authorities,
      jti: 'a610a10-cca6-41db-985f-e87efb303aaf',
      client_id: 'clientid',
    }

    return jwt.sign(payload, 'secret', { expiresIn: '1h' })
  }

  const createResWithToken = ({ authorities }: { authorities: string[] }): Response =>
    createMock<Response>({
      locals: {
        user: {
          token: createToken(authorities),
          roles: [],
        },
      },
    })
  
```

Reply:
Yeah that's a fair spot. I copied the basis of these tests from dashboard controller and didn't even notice the styling change! The methods can be been in the same scope too, just got mixed up along the amount of test tweaking I went through 👍